pkg/controller/migplan/migplan_controller.go

```diff
@@ -157,37 +160,25 @@ func (r *ReconcileMigPlan) Reconcile(request reconcile.Request) (reconcile.Resul
 		}
 	}
 
-	if plan.Status.HasCriticalCondition() {
-		plan.Status.SetReady(false, ReadyMessage)
```

Comment:
I guess the Ready condition isn't meant to be set directly, is that right? I thought it was supposed to be set based on the set of other existing Conditions.

Reply:
Not sure I understand what you're asking.  The `SetReady()` is just for convenience and standardization.  This -/+ is related to setting the ready condition in one place at the end.